pkg/gcv/cf/audit.go

```diff
@@ -25,8 +25,10 @@ audit[result] {
 
 	asset := inventory[_]
 	constraint := constraints[_]
-	
-	re_match(constraint.spec.match.gcp.target[_], asset.ancestry_path)
+	gcp := _get_default(constraint.spec.match, "gcp", {})	
+	# Default matcher behavior is to match everything.
+	target := _get_default(gcp, "target", ["organization/*"])
+	re_match(target[_], asset.ancestry_path)
```

Comment:
Do we expect ancestry_pathto always be provided?
If so can we add a check in add constraint to confirm

Reply:
sure. looks like we're not checking any of the asset fields today? I can do that in a follow-up change to verify the presence of required fields.